modules/tappxBidAdapter.js

```diff
@@ -253,6 +264,25 @@ function buildOneRequest(validBidRequests, bidderRequest) {
   if (config.getConfig('coppa') === true) {
     regs.coppa = config.getConfig('coppa') === true ? 1 : 0;
   }
+
+  // Universal ID
+  const eidsArr = utils.deepAccess(validBidRequests, 'userIdAsEids');
+
+  let eids = [];
+  eidsArr.forEach(eidsElement => {
+    if (eidsElement.source && eidsElement.uids[0].id) {
+      eids.push({
+        source: eidsElement.source,
+        userId: eidsElement.uids[0].id
+      });
+    }
+  });
+
+  let user = {};
+  user.ext = {};
+  user.ext.eids = eidsArr;
+  payload.user = user;
```

Comment:
this could be written in a single line like this:

```suggestion
  payload.user = {
    ext: {
      eids: eidsArr
    }
  };
```

Reply:
Updated in commit 37d9f96